tests/basis-ecp/input.dat

```diff
@@ -0,0 +1,56 @@
+memory 1000 mb
```

Comment:
Are we still adding `#! description` type headers?

Reply:
yes. I usually just wait to catch the missing ones at some supervised docs build, when it prints a list of offenders.

Reply:
okey dokey - just wanted to check in case it was accidentally forgotten